tv_apps/fling-sender/js/fling_sender.js

```diff
@@ -63,13 +67,20 @@
   };
 
   proto.handleMessage = function fs_handleMessage(element, msg) {
-    element.textContent += JSON.stringify(msg);
+    element.textContent = JSON.stringify(msg);
   };
 
-  proto.enableButtons = function fs_enableButtons() {
-    ['load-btn', 'play-btn', 'pause-btn', 'seek-btn',
+  proto.handleStateChange = function fs_handleStateChange() {
+    if (!this._session || this._session.state === 'disconnected') {
+      this.enableButtons(true);
```

Comment:
This reads somewhat weird.. I'd suggest change the function name to `disableButtons` or invert the argument such that the argument specifies whether it "enables" the button or not.